pyomo/contrib/latex_printer/tests/test_latex_printer_vartypes.py

```diff
@@ -1,7 +1,7 @@
 #  ___________________________________________________________________________
 #
 #  Pyomo: Python Optimization Modeling Objects
-#  Copyright (c) 2008-2024
+#  Copyright (c) 2008-2025
 #  National Technology and Engineering Solutions of Sandia, LLC
```

Comment:
The copyright somehow got duplicated in this file (I think it happened last year?)

Reply:
So it did... Fixed!